tests/e2e/vms/xsvm.go

```diff
@@ -0,0 +1,173 @@
+	require := require.New(ginkgo.GinkgoT())
+
+	ginkgo.It("should support transfers between subnets", func() {
+		network := e2e.Env.GetNetwork()
+
+		sourceSubnet := network.GetSubnet(subnetAName)
+		require.NotNil(sourceSubnet)
+		destinationSubnet := network.GetSubnet(subnetBName)
+		require.NotNil(destinationSubnet)
+
+		sourceChain := sourceSubnet.Chains[0]
+		destinationChain := destinationSubnet.Chains[0]
+
+		apiNode := network.Nodes[0]
+		tests.Outf(" issuing transactions on %s (%s)\n", apiNode.NodeID, apiNode.URI)
+
+		destinationKey, err := secp256k1.NewPrivateKey()
+		require.NoError(err)
+
+		ginkgo.By("checking that the funded key has sufficient funds for the export")
+		sourceClient := api.NewClient(apiNode.URI, sourceChain.ChainID.String())
+		initialSourcedBalance, err := sourceClient.Balance(
+			e2e.DefaultContext(),
+			sourceChain.PreFundedKey.Address(),
+			sourceChain.ChainID,
+		)
+		require.NoError(err)
+		require.GreaterOrEqual(initialSourcedBalance, units.Schmeckle)
+
+		ginkgo.By(fmt.Sprintf("exporting from chain %s on subnet %s", sourceChain.ChainID, sourceSubnet.SubnetID))
+		exportTxStatus, err := export.Export(
+			e2e.DefaultContext(),
+			&export.Config{
+				URI:                apiNode.URI,
+				SourceChainID:      sourceChain.ChainID,
+				DestinationChainID: destinationChain.ChainID,
+				Amount:             units.Schmeckle,
+				To:                 destinationKey.Address(),
+				PrivateKey:         sourceChain.PreFundedKey,
+			},
+		)
+		require.NoError(err)
+		tests.Outf(" issued transaction with ID: %s\n", exportTxStatus.TxID)
+
+		ginkgo.By(fmt.Sprintf("issuing transactions on chain %s on subnet %s to activate snowman++ consensus",
+			destinationChain.ChainID, destinationSubnet.SubnetID))
+		recipientKey, err := secp256k1.NewPrivateKey()
+		require.NoError(err)
+		for i := 0; i < numTriggerTxs; i++ {
+			transferTxStatus, err := transfer.Transfer(
+				e2e.DefaultContext(),
+				&transfer.Config{
+					URI:        apiNode.URI,
+					ChainID:    destinationChain.ChainID,
+					AssetID:    destinationChain.ChainID,
+					Amount:     units.Schmeckle,
+					To:         recipientKey.Address(),
+					PrivateKey: destinationChain.PreFundedKey,
+				},
+			)
+			require.NoError(err)
+			tests.Outf(" issued transaction with ID: %s\n", transferTxStatus.TxID)
+		}
+
+		ginkgo.By(fmt.Sprintf("importing to blockchain %s on subnet %s", destinationChain.ChainID, destinationSubnet.SubnetID))
+		sourceURIs := make([]string, len(network.Nodes))
+		for i, node := range network.Nodes {
+			sourceURIs[i] = node.URI
+		}
+		importTxStatus, err := importtx.Import(
+			e2e.DefaultContext(),
+			&importtx.Config{
+				URI:                apiNode.URI,
+				SourceURIs:         sourceURIs,
+				SourceChainID:      sourceChain.ChainID.String(),
+				DestinationChainID: destinationChain.ChainID.String(),
+				TxID:               exportTxStatus.TxID,
+				PrivateKey:         destinationKey,
+			},
+		)
```

Comment:
Technically, I think we need to guarantee that the `export` tx has been accepted on all of the nodes prior to trying to issue the `import`. `export.Export` only waits for the API node to have accepted it.